server.js

```diff
@@ -7,7 +7,9 @@ new WebpackDevServer(webpack(config), {
   contentBase: baseConfig.output.path,
   publicPath: baseConfig.output.publicPath,
   hot: process.env.NODE_ENV === 'development' ? true : false,
-  historyApiFallback: true,
+  historyApiFallback: {
+    disableDotRule: true,
+  },
```

Comment:
> Can we just set historyApiFallback to false?

The previous value was `true` and providing an options object enables it as well; why should it be `false` now?